### DISCUSSION TITLE
Qualifying Card

### DISCUSSION THREAD

**Discussion by brhahlen** [marcokreeft87/formulaone-card#143] [General]
Hi @marcokreeft87 ,

This doesn't warrant a bug, but wanted to check:
In [0.9.0](https://github.com/marcokreeft87/formulaone-card/releases/tag/0.9.0) you created a qualifying results card.
But in 1.0.0 this card is removed again?

Was that intentional?

Thanks for your hard work!

**Comment by marcokreeft87** [comment]
Yes I put it in a tab below the results card. I also added a sprint results card.

**Reply by brhahlen** [reply]
I don't use that card (yet). I'll check it out, thanks again!

**Comment by JunghaJungha** [comment]
Hi Marco,

Thanks for the great work on this.
Is it possible to show the qualifying / sprint results in the results card before the race starts?
Edit: It actually does, but not when selecting other races and then the current one.

Koen

**Reply by marcokreeft87** [reply]
The results card shows this before the race. And thnx!
